client/src/components/modules/Navbar.js

```diff
@@ -207,7 +207,9 @@ function Navbar(props) {
   }
 
   useEffect(() => {
-    setVisible(isIncomplete());
+    if (isIncomplete()) {
```

Comment:
we could add `!user.discordId` to `isIncomplete` if we want to force everyone to link discord, but that might be too intrusive. another option is to require the user to link their discord if they haven't done so already when they register for a tourney. i guess we can see what kasumi thinks

Reply:
yeah, I didn't want to have the user modal start popping up for everyone suddenly. I was planning the next pr to show an error on signup telling the user to link discord